e2e/discussions.spec.ts

```diff
@@ -163,6 +179,8 @@ test.describe('Discussions', () => {
   testAuth(
     'should allow discussion and reply CRUD',
     async ({ userPage, validatedUserPage }) => {
+      test.slow()
```

Comment:
The test intermittently times out, so give the test more time to finish, e.g. when network connection is slow